migrations/1_1_down.sql

```diff
@@ -0,0 +1,7 @@
+DROP INDEX thread_post_deleted_idx;
+DROP TABLE board.thread_post;
+
+DROP INDEX thread_deleted_idx;
+DROP TABLE board.thread;
+
+DROP TABLE "board.user";
```

Comment:
I think I know the reason for this, user is a protected keyword in postgres right, hence the quotes?

Reply:
Yeah, was having some trouble dropping it without the quotes.